### DISCUSSION TITLE
Aligning Human probe gene sequences against Pig reference genome using STAR

### DISCUSSION THREAD

**Discussion by pradeepbkarma** [alexdobin/STAR#1804] [Q&A]
Dear All!
I am trying to align human probe sequences from 10x genomics against the reference Pig reference genome to see if there is a 100% sequence match of human genes with the pig genes and if they have the homology in gene sequences. To perform this alignment, I have some confusion listed below:
1. Can I use the whole reference genome, and annotation GTF file in STAR index or should I use the transcriptome mRNA file?
2. Does STAR find the 100% sequence match?

**Comment by alexdobin** [comment]
Hi @pradeepbkarma 

>Can I use the whole reference genome, and annotation GTF file in STAR index or should I use the transcriptome mRNA file?
Both approaches should work. Since the probes are likely taken from the spliced transcript sequences, mapping to the transcriptome will be more sensitive. 

>Does STAR find the 100% sequence match?

Yes
